# EmbryophyteTEs

This repo contains the project Andrew Lindsay did during his internship under [Dr. Michael Deyholos] (https://biology.ok.ubc.ca/about/contact/michael-deyholos/) with the Irving K. Barber Faculty of Science at the University of British Columbia - Okanagan. Done from 15/05/2017 to 11/08/2017

## Abstract

Transposable elements (TEs) are ubiquitous among organisms, and have shown to have 
significant effect on gene expression and evolution. To best identify these elements, a comparison of existing libraries was performed, as well as a test of software that generates novel TEs.To characterize patterns in both inter and intra-species relationships involving TEs, we analyzed 33 genomes of Embryophytes, and 18 accessions of *Arabidopsis thaliana*. We found relationships between genome size and TE richness, and a similar pattern with genome size and LTR elements. Evidence of TE horizontal gene transfer (HGT) events was observed with the presence of several TE subfamilies unique to a small subset of species, as well as a potential TE extinction event. For Arabidopsis, a potential flaw in sequencing methodology may have been discovered to reduce the divergence of repetitive sequences. Taken together, the results of this study demonstrate the hidden relationships and activities between TEs and the host genome both within a species, and across the evolutionary tree

## Project files
File | Description
------------- | -------------
Scripts | Assorted R, Python and Bash scripts used in the analysis 
Images | Selected plots as examples
Obs of TEs - Thesis Presentation.pdf | Final presentation
Obs of TEs - Thesis Report.pdf | Final report
TE Literature Review.pdf | Literature review of current knowledge for empryophyte transposable elements 

## Images

<table border="0" >
   <tr>
    <td>
  <img src="https://github.com/knacko/EmbryophyteTEs/blob/main/Images/TElandscape.png" width="600"> 
    </td>
    <td>
  <sub><b>Figure 12. Diversity of TE classes as a percentage of the genome and of TE total length.</b> Species in the same genus are grouped in orange 
boxes. SINEs are present on both graphs, but are indistinguishable in the top, and limited to the lower edge of the bottom graph</sub>  
    </td>
  </tr>
  <tr>
    <td>
  <img src="https://github.com/knacko/EmbryophyteTEs/blob/main/Images/viri-subfamilies.jpg" width="600"> 
    </td>
    <td>
  <sub><b>Figure 13. Subfamily richness for Embryophyte genomes.</b> TE subfamilies content was determined as a percentage of the genome, using TAIR
as the masking library. Subfamilies binned as 0% show no elements in the genome. The dendrogram was derived with complete-linkage 
hierarchical clustering based on true proportion, rather than binned data. Species with a common genus are highlighted in boxes.</sub>
  </tr>
  </table>

